lesson_02/job1/dl/extract.py

```diff
@@ -0,0 +1,27 @@
+import requests
+import os
+import sys
+if __name__ == '__main__':
+    current_dir=os.path.dirname(os.path.abspath(__file__))
+    sys.path.append(os.path.abspath(os.path.join(current_dir,'..','..')))
+    from env.env_variables import get_env_variable
+else:
+    from env.env_variables import get_env_variable
+def request(period,page):
+    custom_ca_cert_path = r'C:\Users\iuar0081\Desktop\Power BI APP\JiraCoonection\rbua 1.pem'
```

Comment:
not need to hardcode local path

Reply:
without this my code is not working in my working laptop. but i understand